mytimer/functions.py

```diff
@@ -368,6 +370,13 @@ def countup_timer(
                     timer_second),
                 font=face,
                 sep="\n" + " " * h_shift)
+        current_time = datetime_now.strftime(CLOCK_FORMAT)
+        current_date = datetime_now.strftime(DATE_FORMAT)
+        print(" " * h_shift, end='')
+        print(current_time)
+        print(" " * h_shift, end='')
+        print(current_date)
+        print("")
```

Comment:
I think you can make it into a separate function.

Reply:
Absolutely, you're correct. I plan to refactor this section in a separate pull request.